### PR TITLE
Changes required to display data from TPC

### PR DESCRIPTION
**Pull request by jmyrcha**

### CODE REVIEW

**Comment by jmyrcha** [comment]
It looks, that it did not build not because EventVisualisation - how we can retry a build/O2/o2 test?

**Comment by sawenzel** [comment]
Can you please fix the codechecker errors:
```
rTPC.h:29:9: error: field declaration 'fMaxEv' does not match naming rule [aliceO2-member-name]
  Int_t fMaxEv;
        ^~~~~~~
        mMaxEv
/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/build_o2checkcode_o2/0/EventVisualisation/Detectors/src/DataReaderTPC.cxx:44:3: error: field reference 'fMaxEv' does not match naming rule [aliceO2-member-name]
  fMaxEv = trackBuffer->size();
  ^~~~~~
  mMaxEv
/mnt/mesos/sandbox/sandbox/sw/SOURCES/O2/build_o2checkcode_o2/0/EventVisualisation/Detectors/src/DataReaderTPC.cxx:49:10: error: field reference 'fMaxEv' does not match naming rule [aliceO2-member-name]
  return fMaxEv;
         ^~~~~~~
         mMaxEv
```
A new build will be triggered upon a code change in any case.

File: EventVisualisation/Detectors/include/EventVisualisationDetectors/DataReaderTPC.h [hunk 1]
```diff
@@ -26,9 +26,15 @@ namespace event_visualisation
...
  private:
+<<<<<<< HEAD:EventVisualisation/Detectors/include/EventVisualisationDetectors/DataReaderTPC.h
+  Int_t mMaxEv;
+  TFile* mClusFile;
+  TFile* mTracFile;
+=======
   Int_t fMaxEv;
   TFile* clusFile;
   TFile* tracFile;
+>>>>>>> f2dea92bebb772f1eb0800290b722a2a284eb318:EventVisualisation/Detectors/include/EventVisualisationDetectors/DataReaderTPC.h
```
Comment (sawenzel) [inline EventVisualisation/Detectors/include/EventVisualisationDetectors/DataReaderTPC.h]: Leftover from merge. This needs to be fixed.

File: EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationCluster.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+class VisualisationCluster
+{
+ public:
+  // Default constructor
+  VisualisationCluster(double XYZ[]);
+
+  double X() { return mCoordinates[0]; }
+  double Y() { return mCoordinates[1]; }
+  double Z() { return mCoordinates[2]; }
```
Comment (sawenzel) [inline EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationCluster.h]: in general, accessor functions should be `const` annotated. This applies to many functions/changes in this PR

File: EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  VisualisationEvent(int eventNumber, int runNumber, double energy, int multiplicity, std::string collidingSystem, time_t timeStamp);
+
+  // Adds visualisation track inside visualisation event
+  void addTrack(const VisualisationTrack& track) { mTracks.push_back(track); }
+
+  // Adds visualisation cluser inside visualisation event
+  void addCluster(const VisualisationCluster& cluster) { mClusters.push_back(cluster); }
+
+  // Multiplicity getter
+  inline int GetMultiplicity() { return mMultiplicity; }
```
Comment (sawenzel) [inline EventVisualisation/DataConverter/include/EventVisualisationDataConverter/VisualisationEvent.h]: no need to put `inline` keyword; please annotate `const`

File: EventVisualisation/DataConverter/src/VisualisationEvent.cxx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                                       string collidingSystem,
+                                       time_t timeStamp) : mEventNumber(eventNumber),
+                                                           mRunNumber(runNumber),
+                                                           mEnergy(energy),
+                                                           mMultiplicity(multiplicity),
+                                                           mCollidingSystem(collidingSystem),
+                                                           mTimeStamp(timeStamp)
+{
+}
+
```
Comment (sawenzel) [inline EventVisualisation/DataConverter/src/VisualisationEvent.cxx]: trivial (one line) accessor such as getTrack / getCluster should be provided in the header

**Review by sawenzel — CHANGES_REQUESTED** [review]
I suggest to improve the code concerning const correctness and inlining small functions.
Would you also be able to attach a screenshot of some TPC data displayed?

**Review by sawenzel — APPROVED** [review]

**Pull request merged**